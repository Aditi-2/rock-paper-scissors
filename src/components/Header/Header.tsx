import React from "react";
import styled from "@emotion/styled";

const Header: React.FC = () => {
  return (
    <HeaderStyled>
      <span className="header-title">Waste an hour having fun</span>
    </HeaderStyled>
  );
};

const HeaderStyled = styled.div`
  background-color: #eaeaea;
  color: #141942;
  position: fixed;
  top: 0px;
  left: 0px;
  width: 100%;
  padding: 1.5rem 1rem;
  -webkit-box-shadow: 0px 8px 15px -3px #000000;
  box-shadow: 0px 8px 15px -3px #000000;

  & .header-title {
    font-size: 1.3rem;
  }
`;

export default Header;
