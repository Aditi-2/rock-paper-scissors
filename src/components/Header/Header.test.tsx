import React from 'react';
import {create} from 'react-test-renderer';
import { render, screen } from '@testing-library/react';
import Header from './Header';

const getComponent = () => <Header />

test('Header Component snapshot', () => {
    const component = create(getComponent()).toJSON();
    expect(component).toMatchSnapshot();
});

test('renders Header Component', () => {
  render(getComponent());
  const headerText = screen.getByText(/Waste an hour having fun/i);
  expect(headerText).toBeInTheDocument();
});

