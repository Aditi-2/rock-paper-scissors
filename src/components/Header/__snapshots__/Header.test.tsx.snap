// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Header Component snapshot 1`] = `
<div
  className="css-1t6ajz4"
>
  <span
    className="header-title"
  >
    Waste an hour having fun
  </span>
</div>
`;
